fcrepo-event-serialization/src/test/java/org/fcrepo/event/serialization/FedoraEventTest.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2015 DuraSpace, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.event.serialization;
+
+import static com.hp.hpl.jena.datatypes.xsd.XSDDatatype.XSDdateTime;
+import static com.hp.hpl.jena.rdf.model.ResourceFactory.createProperty;
+import static com.hp.hpl.jena.rdf.model.ResourceFactory.createResource;
+import static com.hp.hpl.jena.rdf.model.ResourceFactory.createTypedLiteral;
+import static com.hp.hpl.jena.vocabulary.DCTerms.identifier;
+import static com.hp.hpl.jena.vocabulary.DCTerms.isPartOf;
+import static com.hp.hpl.jena.vocabulary.RDF.type;
+import static java.time.Instant.ofEpochMilli;
+import static org.fcrepo.kernel.api.RdfLexicon.EVENT_NAMESPACE;
+import static org.fcrepo.kernel.api.RdfLexicon.PROV_NAMESPACE;
+import static org.fcrepo.kernel.api.RdfLexicon.REPOSITORY_NAMESPACE;
+import static org.fcrepo.kernel.api.observer.OptionalValues.BASE_URL;
+import static org.fcrepo.kernel.api.observer.OptionalValues.USER_AGENT;
+import static org.fcrepo.event.serialization.EventSerializer.toModel;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.when;
+
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import com.hp.hpl.jena.rdf.model.Model;
+import com.hp.hpl.jena.rdf.model.Resource;
+import com.hp.hpl.jena.rdf.model.SimpleSelector;
+import org.fcrepo.kernel.api.observer.EventType;
+import org.fcrepo.kernel.api.observer.FedoraEvent;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+/**
+ * <p>FedoraEventTest class.</p>
+ *
+ * @author acoburn
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class FedoraEventTest {
```

Comment:
These class names may well be a result of splitting and combining maven modules, but this test leans towards the `TurtleSerializer`, particularly if `EventSerializer.toModel()` is moved to `TurtleSerializer`. 
Naming this class `FedoraEventTest` seems to give it greater emphasis... particularly since the default "Fedora Events" are serialized with the `JsonLDSerializer`.